.github/workflows/r.yml

```
@@ -275,6 +275,19 @@ jobs:
           working-directory: 'r'
           extra-packages: |
             any::rcmdcheck
+      - name: Install MinIO
+        shell: bash
+        run: |
+          mkdir -p "$HOME/.local/bin"
+          curl \
+            --output "$HOME/.local/bin/minio.exe" \
+            https://dl.min.io/server/minio/release/windows-amd64/archive/minio.RELEASE.2022-05-26T05-48-41Z
+          chmod +x "$HOME/.local/bin/minio.exe"
+          echo "$HOME/.local/bin" >> $GITHUB_PATH
+      # TODO: figure out why the GCS tests are hanging
+      # - name: Install Google Cloud Storage Testbench
+      #   shell: bash
+      #   run: ci/scripts/install_gcs_testbench.sh default
```

Is this something you're trying to solve in this PR?

---

Ah I had created this earlier https://issues.apache.org/jira/browse/ARROW-17149

```suggestion
      # TODO(ARROW-17149): figure out why the GCS tests are hanging on Windows
      # - name: Install Google Cloud Storage Testbench
      #   shell: bash
      #   run: ci/scripts/install_gcs_testbench.sh default
```